.travis.yml

```diff
@@ -42,5 +42,6 @@ after_success:
 branches:
   only:
     - master
+    - dev-v1
```

Comment:
You might want to add a comment explaining that this is temporary and not to forget to remove it.

Reply:
Good point, thanks.